io.github.ferraridamiano.ConverterNOW.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
did you submit this file upstream? I GUI application that doesn't install a .desktop, it's a bug as it is an important component.

Reply:
I'm sorry but I don't understand. This file is included in the PR, hence it is submitted

Reply:
*UPSTREAM*